cli/azd/pkg/apphost/generate.go

```diff
@@ -349,7 +349,12 @@ func (b *infraGenerator) LoadManifest(m *Manifest) error {
 		case "azure.sql.v0", "sqlserver.server.v0":
 			b.addSqlServer(name)
 		case "azure.sql.database.v0", "sqlserver.database.v0":
-			b.addSqlDatabase(*comp.Parent, name)
+			if comp.Parent == nil ||
```

Comment:
I wonder if it's better to blow up if `parent == nil` then to ignore?

Reply:
I think it should be an error,,,  I was thinking about it yesterday and update some parts to return an error if parent is not defined for a db... that'd be an error